### ISSUE TITLE
Feature Request: Stacked bar chart

### ISSUE THREAD

**Issue by michaelblight** [kalkih/mini-graph-card#79]
I'm plotting CPU usage of my 12 Docker containers, which are percentages. It would therefore be really useful to stack the graphs so the overall CPU usage would also be shown. The alternative is to fudge it by keeping a running total myself (ie. the last container's CPU is a sum of it and all the previous), but then you can't see valid values for each data point.

**Comment by kalkih** [comment]
Interesting idea, you mean something like this? 

![5-stacked-bar-chart-large-opt](https://user-images.githubusercontent.com/457678/55573564-973a9880-570a-11e9-8b8c-57cc47e7f290.png)

Might be a bit out of scope / too advanced for this mini graph project, but definitely a cool and useful feature.

You could create a template sensor to easily get the total or checkout [systemmonitor](https://www.home-assistant.io/components/systemmonitor/), and then show it in a seperate graph. Not an ideal solution but works.

**Comment by michaelblight** [comment]
Yes, that's it. I was hoping it wouldn't be too hard on the assumption that the underlying graph was a component you were using rather than one you wrote. I was going to try my own fudge - the main downside is hovering over one of the graphs will show you the cumulative total, not it's actual value.

**Comment by bdr99** [comment]
This would be great for power consumption graphs. By stacking the lines, you would be able to see both the total power consumption as well as how much each individual device contributes to the total.

Something like this:
![image](https://user-images.githubusercontent.com/2292715/70743633-2cf0e380-1cee-11ea-8fca-f27ab62a699b.png)

**Comment by alexswart** [comment]
this is what i am looking for to stack 2 value's in 1 bar
is there a solution to show it? i want see my energy peak and off peak in 1 bar with 2 different collors

**Comment by firstof9** [comment]
Would it be possible to do a mode like this image but vertical?
![titanic-age-pyramid-1](https://user-images.githubusercontent.com/1105672/78203901-0b5a6a00-744d-11ea-9da5-83ef4af6b2ff.png)

**Comment by da-anda** [comment]
I would also love to see some sort of stacking feature. Like basically creating a new virtual entity in which you define the `aggregate_func` to your liking, like `sum` for stacked stuff.

**Comment by doppiej** [comment]
I would also love to see the stacking feature, both for line graphs and bar graphs!
Is there a possibility this will be added in the (near) future?
Thanks in advance and I really like using these cards!

**Comment by Toukite** [comment]
Hello, I would also like to have this stacking feature !

**Comment by ohipe** [comment]
hi @kalkih do you have any update on this? is it on your roadmap?
thanks for your work it is amazing!

**Comment by Stefan4810** [comment]
?

**Comment by RomRider** [comment]
Unlikely to happen in `mini-graph-card` however this feature is available in `apexcharts-card` now:
https://github.com/RomRider/apexcharts-card

We'll keep this one `mini` and the other one will have more advanced features.

**Comment by MayeulC** [comment]
Could this be reopened? I am very much interested in such a feature myself.

I have sensors for individual devices' power consumption, as well as total power. It would be nice to have total power in the background, and individual devices stacked in the foreground.

Since individual devices usually have similar power ratings (~30W), they can be hard to distinguish. Such a graph would also tell me at a glance how much power is unaccounted for.

**Comment by ildar170975** [comment]
Implemented here: https://github.com/kalkih/mini-graph-card/pull/1376#issuecomment-4930477941

<img width="485" height="311" alt="Image" src="https://github.com/user-attachments/assets/ce147a4c-223f-43b2-bad9-1152b5cbde34" />

Not exactly a sum, not exactly “stacked” (when a next bar starts where a previous bar ends), just a similar look which I called “overlapping bars”.
